lib/bolt/result.rb

```diff
@@ -95,12 +119,15 @@ def to_h
     private
 
     def output_to_json_hash(output)
-      obj = JSON.parse(output)
-      if obj.is_a? Hash
-        obj
+      begin
+        obj = JSON.parse(output)
+        if obj.is_a? Hash
+          obj
```

Comment:
This won't be returned, so the whole if-clause is pointless. Not sure this is doing what you intend because if `obj` is json but not a hash, we're still going to reach line 130 and return obj rather than wrap it in `_output`.

Reply:
Should add a test covering that case after you decide how it should be handled.

Reply:
Running `bolt command run 'echo "{"a": 1}"' -n node1` didn't do what I'd expect. Piped through `jq` it looks like 
```
{
  "items": [
    {
      "name": "node1",
      "status": "success",
      "result": {
        "stdout": "{a: 1}\n",
        "stderr": "",
        "exit_code": 0
      }
    }
  ],
  "node_count": 1,
  "elapsed_time": 0
}
```

I thought it would be parsed in some way.